diff --git a/build-docker.sh b/build-docker.sh
index 01fd517..5bbad07 100755
--- a/build-docker.sh
+++ b/build-docker.sh
@@ -109,6 +109,8 @@ else
 		-v /lib/modules:/lib/modules \
 		${PIGEN_DOCKER_OPTS} \
 		--volume "${CONFIG_FILE}":/config:ro \
+		--volume "${DIR}/work":/pi-gen/work \
+		--volume "${DIR}/deploy":/pi-gen/deploy \
 		-e "GIT_HASH=${GIT_HASH}" \
 		pi-gen \
 		bash -e -o pipefail -c "dpkg-reconfigure qemu-user-static &&
@@ -118,7 +120,7 @@ else
 fi
 
 echo "copying results from deploy/"
-${DOCKER} cp "${CONTAINER_NAME}":/pi-gen/deploy .
+#${DOCKER} cp "${CONTAINER_NAME}":/pi-gen/deploy .
 ls -lah deploy
 
 # cleanup
diff --git a/stage2/01-sys-tweaks/00-packages b/stage2/01-sys-tweaks/00-packages
index a1f4b59..00fbe74 100644
--- a/stage2/01-sys-tweaks/00-packages
+++ b/stage2/01-sys-tweaks/00-packages
@@ -1,33 +1,16 @@
 ssh less fbset sudo psmisc strace ed ncdu crda
 console-setup keyboard-configuration debconf-utils parted unzip
-build-essential manpages-dev python bash-completion gdb pkg-config
-python-rpi.gpio v4l-utils
-python-gpiozero
-python3-gpiozero
 avahi-daemon
-lua5.1
-luajit
 hardlink ca-certificates curl
-fake-hwclock nfs-common usbutils
-libraspberrypi-dev libraspberrypi-doc libfreetype6-dev
-dosfstools
-dphys-swapfile
+fake-hwclock usbutils
 raspberrypi-sys-mods
 pi-bluetooth
-apt-listchanges
-usb-modeswitch
 libpam-chksshpwd
 rpi-update
-libmtp-runtime
-rsync
-htop
-man-db
-policykit-1
 ssh-import-id
 rng-tools
 ethtool
 vl805fw
-ntfs-3g
 pciutils
 rpi-eeprom
 raspinfo
diff --git a/stage2/01-sys-tweaks/00-packages-nr b/stage2/01-sys-tweaks/00-packages-nr
deleted file mode 100644
index 5acef54..0000000
--- a/stage2/01-sys-tweaks/00-packages-nr
+++ /dev/null
@@ -1 +0,0 @@
-cifs-utils
diff --git a/stage2/01-sys-tweaks/00-patches/02-swap.diff b/stage2/01-sys-tweaks/00-patches/02-swap.diff
deleted file mode 100644
index 745a344..0000000
--- a/stage2/01-sys-tweaks/00-patches/02-swap.diff
+++ /dev/null
@@ -1,13 +0,0 @@
-Index: jessie-stage2/rootfs/etc/dphys-swapfile
-===================================================================
---- jessie-stage2.orig/rootfs/etc/dphys-swapfile
-+++ jessie-stage2/rootfs/etc/dphys-swapfile
-@@ -13,7 +13,7 @@
- 
- # set size to absolute value, leaving empty (default) then uses computed value
- #   you most likely don't want this, unless you have an special disk situation
--#CONF_SWAPSIZE=
-+CONF_SWAPSIZE=100
- 
- # set size to computed value, this times RAM size, dynamically adapts,
- #   guarantees that there is enough swap without wasting disk space on excess
diff --git a/stage2/01-sys-tweaks/00-patches/series b/stage2/01-sys-tweaks/00-patches/series
index aee0402..c5d8e4b 100644
--- a/stage2/01-sys-tweaks/00-patches/series
+++ b/stage2/01-sys-tweaks/00-patches/series
@@ -1,5 +1,4 @@
 01-useradd.diff
-02-swap.diff
 04-inputrc.diff
 05-path.diff
 07-resize-init.diff
diff --git a/stage2/01-sys-tweaks/01-run.sh b/stage2/01-sys-tweaks/01-run.sh
index c1836f9..9a3672a 100755
--- a/stage2/01-sys-tweaks/01-run.sh
+++ b/stage2/01-sys-tweaks/01-run.sh
@@ -25,8 +25,8 @@ fi
 
 on_chroot << EOF
 systemctl disable hwclock.sh
-systemctl disable nfs-common
-systemctl disable rpcbind
+#systemctl disable nfs-common
+#systemctl disable rpcbind
 if [ "${ENABLE_SSH}" == "1" ]; then
 	systemctl enable ssh
 else
diff --git a/stage2/02-net-tweaks/00-packages b/stage2/02-net-tweaks/00-packages
index cc4a68e..0e36e7a 100644
--- a/stage2/02-net-tweaks/00-packages
+++ b/stage2/02-net-tweaks/00-packages
@@ -1,4 +1,3 @@
-wpasupplicant wireless-tools firmware-atheros firmware-brcm80211 firmware-libertas firmware-misc-nonfree firmware-realtek
 raspberrypi-net-mods
 dhcpcd5
 net-tools
diff --git a/stage2/03-accept-mathematica-eula/00-debconf b/stage2/03-accept-mathematica-eula/00-debconf
deleted file mode 100644
index d9743fe..0000000
--- a/stage2/03-accept-mathematica-eula/00-debconf
+++ /dev/null
@@ -1,2 +0,0 @@
-# Do you accept the Wolfram - Raspberry Pi® Bundle License Agreement?
-wolfram-engine  shared/accepted-wolfram-eula    boolean true
diff --git a/stage2/EXPORT_IMAGE b/stage2/EXPORT_IMAGE
deleted file mode 100644
index aa8f936..0000000
--- a/stage2/EXPORT_IMAGE
+++ /dev/null
@@ -1,4 +0,0 @@
-IMG_SUFFIX="-lite"
-if [ "${USE_QEMU}" = "1" ]; then
-	export IMG_SUFFIX="${IMG_SUFFIX}-qemu"
-fi
diff --git a/stage2/EXPORT_NOOBS b/stage2/EXPORT_NOOBS
deleted file mode 100644
index 0d78ecf..0000000
--- a/stage2/EXPORT_NOOBS
+++ /dev/null
@@ -1,2 +0,0 @@
-NOOBS_NAME="Raspberry Pi OS Lite (32-bit)"
-NOOBS_DESCRIPTION="A port of Debian with no desktop environment"
